src/librustdoc/doctest.rs

```diff
@@ -110,8 +110,6 @@ crate fn run(options: Options) -> Result<(), ErrorReported> {
 
     let res = interface::run_compiler(config, |compiler| {
         compiler.enter(|queries| {
-            let _lower_to_hir = queries.lower_to_hir()?;
-
             let crate_name = queries.crate_name()?.peek().to_string();
             let mut global_ctxt = queries.global_ctxt()?.take();
```

Comment:
unrelated, but this seems unnecessarily complicated, it could just call `tcx.crate_name()` below. I'll make a PR for that.

Reply:
https://github.com/rust-lang/rust/pull/86501